README.rst

```diff
@@ -274,12 +274,43 @@ as devices with the following command.
 
    zenbatchload <filename>
 
+Modeling and Monitoring OS Services
+-----------------------------------
+The Linux OS services are modeled using the *zenoss.cmd.linux.os_service* modeler plugin. The following systems are supported:
+
+- systemd (RHEL 7)
+- upstart (RHEL 6)
+- systemV (RHEL 5 and earlier)
+
+Version 2.3.0 and beyond provides support for monitoring the status of the services every collection cycle. However, only **systemd** and **systemV** are supported for monitoring. The following two zProperties filter the services that are to be modeled and thereby monitored.
```

Comment:
Replace the entire paragraph with :

with

`Version 2.3.0  supports monitoring of the status of **systemd** and **systemV** system services. The zProperties 
*zLinuxServicesModeled* and *zLinuxServicesNotModeled* restrict the services that are modeled and monitored:`